CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+## [v1.4.2](https://github.com/76creates/stickers/compare/v1.4.1...v1.4.2) (2024-11-26)
+### ⚠ BREAKING CHANGES
+- Fix typo in public func identifiers (replace Heigth with Height).
```

Comment:
You don't have to make it a breaking changes yet

See https://github.com/76creates/stickers/pull/18#discussion_r1859187666